scripts/collate_packages.py

```diff
@@ -1,21 +1,22 @@
+""" Collate packages """
+
 import os
 import sys
-import git
 
 import scriptutils
 
-error = False
+ERROR = False
```

Comment:
uppercase is usually used for constants, but this is not a constant, so I think the rule doing this should be disabled.

Reply:
> It looks like there is still an UPPERCASE on script variables - I don't see a rule that's doing that, though; was that done by hand somehow?
> 
> Some of these that you just disabled I think are probably fine to turn back on (though you don't have to):
> 
>     * E226 missing whitespace around arithmetic operator
> 
>     * E231 missing whitespace after ','
> 
>     * E251 unexpected spaces around keyword / parameter equals
> 
>     * E265 block comment should start with '# '
> 
>     * E302 expected 2 blank lines, found 1
> 
>     * E305 expected 2 blank lines after class or function definition, found 1
> 
>     * W291 trailing whitespace

My bad, looking back at the commits history, it turns out that I did do some edits manually. I go through them and correct them.